src/metabase/driver/crate/native.clj

```diff
@@ -0,0 +1,31 @@
+(ns metabase.driver.crate.native
+  (:require [clojure.java.jdbc :as jdbc]
+            [metabase.models.database :refer [Database]]
+            [metabase.db :refer [sel]]
+            [metabase.driver.generic-sql :as sql]
+            [clojure.tools.logging :as log]
+            [metabase.util :as u]
+            [metabase.driver.generic-sql.native :as n]))
+
+(defn process-and-run
```

Comment:
what is the reason behind duplicating this function instead of using the one in `generic-sql/native` ?